docs/sources/get-started/quick-start.md

```diff
@@ -2,160 +2,248 @@
 
-    1. View all the log lines which have the container label "flog":
+   After copying any of these queries into the query editor, click **Run Query** (4) to execute the query.
 
-        ```bash
-        {container="evaluate-loki-flog-1"}
-        ```
+   1. View all the log lines which have the container label `evaluate-loki-flog-1`:
 
-        In Loki, this is called a log stream. Loki uses [labels](https://grafana.com/docs/loki/<LOKI_VERSION>/get-started/labels/) as metadata to describe log streams.  Loki queries always start with a label selector.  In the query above, the label selector is `container`.
+      <!-- Killercoda copy START -->
 
-    1. To view all the log lines which have the container label "grafana":
+      ```bash
+      {container="evaluate-loki-flog-1"}
+      ```
 
-        ```bash
-        {container="evaluate-loki-grafana-1"}
-        ```
+      <!-- Killercoda copy END -->
 
-    1. Find all the log lines in the container=flog stream that contain the string "status":
+      In Loki, this is a log stream.
 
-        ```bash
-        {container="evaluate-loki-flog-1"} |= `status`
-        ```
+      Loki uses [labels](https://grafana.com/docs/loki/<LOKI_VERSION>/get-started/labels/) as metadata to describe log streams.
 
-    1. Find all the log lines in the container=flog stream where the JSON field "status" is "404":
+      Loki queries always start with a label selector.
+      In the previous query, the label selector is `{container="evaluate-loki-flog-1"}`.
 
-        ```bash
-        {container="evaluate-loki-flog-1"} | json | status=`404`
-        ```
+   1. To view all the log lines which have the container label `evaluate-loki-grafana-1`:
 
-    1. Calculate the number of logs per second where the JSON field "status" is "404":
+      <!-- Killercoda copy START -->
 
-        ```bash
-        sum by(container) (rate({container="evaluate-loki-flog-1"} | json | status=`404` [$__auto]))        
-        ```
+      ```bash
+      {container="evaluate-loki-grafana-1"}
+      ```
 
-    The final query above is a metric query which returns a time series. This will trigger Grafana to draw a graph of the results.  You can change the type of graph for a different view of the data.  Click **Bars** to view a bar graph of the data.
+      <!-- Killercoda copy END -->
 
-1. Click the **Builder** tab (3) to return to Builder mode in the query editor.
-    1. In Builder view, click **Kick start your query**(5).
-    1. Expand the **Log query starters** section.
-    1. Select the first choice, **Parse log lines with logfmt parser**, by clicking **Use this query**.
-    1. On the Explore tab, click **Label browser**, in the dialog select a container and click **Show logs**.
+   1. Find all the log lines in the `{container="evaluate-loki-flog-1}` stream that contain the string `status`:
+
+      <!-- Killercoda copy START -->
+
+      ```bash
+      {container="evaluate-loki-flog-1"} |= `status`
```

Comment:
So this is more of a me problem than yours but thought I would mention it here to see what we could do. The Docker compose on within the Killercoda instance is slightly more dated than that of the Docker Desktop instances. So they represent the container names slightly differently. 

Docker Desktop:
```json
{container="evaluate-loki-flog-1"} |= `status`
```

Docker on Ubuntu 20.04
```
{container="evaluate_loki-flog-1"} |= `status`
```
Stupid I know! Do you think its possible to add render only in Killercoda note for instance:
```
<!-- Killercoda Note START -->
This docker issue ruins my day
<!-- Killercoda Note END -->

```

Reply:
We definitely should think about comments for including Killercoda only content.

I think it would all have to be HTML comments so it doesn't render in the website page. Something like:

```markdown
<!-- Killercoda include START -->
<!-- This Docker issue ruins my day -->
<!-- Killercoda include END -->
```

For the specific issue of the container names. Can we make it consistent for both by specifying the `container_name` property on each container in the Docker Compose file?